client/src/components/LoginForm/LoginContent.jsx

```diff
@@ -1,19 +1,120 @@
 import React, { Component } from 'react';
 import Section from '../Section';
 import EditText from '../EditText';
+import axios from 'axios';
 import './LoginForm.css';
 
+const LOGIN_URL = 'http://localhost:3001/users/login';
+
 class LoginContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.onLogin = this.onLogin.bind(this);
+  }
+
+  onLogin() {
+    if (this.isInputValid()) {
+      //Send Request
+      axios
+        .post(LOGIN_URL, {
+          username: this.state.username,
+          password: this.state.password,
+        })
+        .then(response => {
+          // Got good response
+          console.log(response);
+          // TODO: change page, save student ID and name when backend is done
+          // temp for testing
+          this.setError('login successful');
+        })
+        .catch(loginError => {
+          // Invalid login
+          console.log(loginError);
+          var error = loginError.response.data.ErrorMessage;
+          if (
+            error === 'Password length must be between 8 and 64' ||
```

Comment:
Theres also: 
"Incorrect username or password."
"Username length must be less than 64"
That would all kind of count as invalid credentials. The numbers are subject to change though (like we can suddenly just decide to put the max username length to 20 for example). So it may be simpler to just display the full error message instead but up to you.

Reply:
The errors will be displayed in `Sign up`, for `Log in` it prevents people from attempting to guess credentials.